torch_ort_inference/docs/install.md

```diff
@@ -0,0 +1,33 @@
+# Install torch-ort-inference
+
+You can install and run torch-ort-inference in your local environment.
+
+## Run in a Python environment
+
+### Default dependencies
+
+By default, torch-ort-inference depends on PyTorch 1.12 and ONNX Runtime OpenVINO EP 1.11.
+
+Install torch-ort-inference
+
+- `pip install torch-ort-inference`
+
+## Test your installation
+
+Once you have created your environment, using Python, execute the following steps to validate that your installation is correct.
+
+1. Download a inference script
+
+    `wget https://raw.githubusercontent.com/pytorch/ort/main/torch_ort_inference/tests/bert_for_sequence_classification.py`
+
+2. Install extra dependencies
+
+    `pip install wget pandas sklearn transformers`
+
+3. Run the inference script
+
+    `python ./ort/torch_ort_inference/tests/bert_for_sequence_classification.py`
+
+4. Expected warnings
+
+   There are some warnings that are expected.
```

Comment:
Should we also add "Build from source" section here and describe how to build ONNX Runtime from source with the relevant execution provider? We should definitely add this in ONNX Runtime docs, but just wondering if we should add here too?

Reply:
Since torch/ort is an abstract layer and onnxruntime-openvino is added as dependency, would be good to maintain the parity with the existing torch-ort.